1-mandatory/1-writers.js

```diff
@@ -62,19 +63,22 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  let writerNalive40 = writers.filter(writer => !writer.alive && 40< writer.age && writer.age<50);
+writerNalive40.forEach(writer=>
+  console.log(`Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`))
 }
 
+
 /*
 Exercise 3:
   Only `console.log()` out alive writers who are in their 40s (meaning between 40 and 49):
   "Hi, my name is {firstName} {lastName}. I am {age} years old."
 */
 
 function logAliveWritersInTheirForties() {
-  // write your code here
+let writerAlive40 = writers.filter(writer => writer.alive &&40<= writer.age && writer.age<50)
```

Comment:
Super minor nitpick

writerAlive40 is `singular` but we are returning an array. Why not `writersAliveIn40s`